Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/main/TvMainActivity.java

```diff
@@ -54,8 +54,8 @@ protected void onCreate(Bundle savedInstanceState)
 		// Stuff in this block only happens when this activity is newly created (i.e. not a rotation)
 		if (savedInstanceState == null)
 			StartupHandler.HandleInit(this);
-		// Setup and/or sync channels
-        TvUtil.scheduleSyncingChannel(getApplicationContext());
+		  // Setup and/or sync channels
+      TvUtil.scheduleSyncingChannel(getApplicationContext());
```

Comment:
This isn't the correct indentation. Instead of increasing the indentation of the comment, you need to decrease the indentation of the line that comes afterwards.

Reply:
Wouldn't it be nice if we had something like clang-format, except for Java?
[Oh wait, I believe we do.](/dolphin-emu/dolphin/blob/master/Source/Android/code-style-java.jar) Or at least something in that vein.

Reply:
Fixed.